src/Logger.spec.ts

```diff
@@ -1,50 +1,62 @@
-import { Logger, LoggerFactory } from '..';
+import { Logger, LoggerFactory } from './';
 
 describe('Logger', () => {
-  let _log = undefined;
-  before(() => {
-    _log = Logger.getLogger(__filename);
+  let log: Logger;
+  beforeAll(() => {
+    log = Logger.getLogger(__filename);
   });
   afterEach(() => {
     LoggerFactory.setFactory(null);
   });
 
   describe('#log', () => {
     it('should exist', () => {
-      _log.log('debug', 'should exist');
+      if (!('log' in log)) {
+        fail();
+      }
+      log.log('debug', 'should exist');
     });
     it('should queue if no factory is installed', (done) => {
       Logger.getLogger(__filename);
-      _log.debug('aloha');
+      log?.debug('aloha');
```

Comment:
This will not fail the test if `debug` does not exist. What's the intention of this line?